### DISCUSSION TITLE
wit-bindgen support for Grain

### DISCUSSION THREAD

**Discussion by ospencer** [grain-lang/grain#1650] [General]
Work has begun on adding support for Grain to https://github.com/bytecodealliance/wit-bindgen. The work is happening here: https://github.com/ospencer/witx-bindgen/tree/oscar/grain-bindgen

That fork is a bit out of date, but it can produce bindings for Grain v0.5.x.

Getting this finished will likely be a priority after v0.6 has landed.

Updates will be posted here!

**Comment by philpax** [comment]
Excited for this! I'd love to see Grain as a first-class guest language for [the project I work on](https://github.com/AmbientRun/Ambient/) 🙂
